src/components/ContentNode.vue

```diff
@@ -232,9 +231,9 @@ function renderNode(createElement, references) {
     case BlockType.heading: {
       const props = {
         anchor: node.anchor,
-        tag: `h${node.level}`,
+        level: Number(node.level),
```

Comment:
I believe that `node.level` values are guaranteed to be integers in the RenderNode spec, so you can just pass `node.level` here without the `Number()` call.

Reply:
fixed!